Analysis/DataModel/include/Analysis/HFSecondaryVertex.h

```diff
@@ -107,6 +107,7 @@ DECLARE_SOA_DYNAMIC_COLUMN(P, p, [](float px, float py, float pz) { return RecoD
 DECLARE_SOA_DYNAMIC_COLUMN(P2, p2, [](float px, float py, float pz) { return RecoDecay::P2(px, py, pz); });
 DECLARE_SOA_DYNAMIC_COLUMN(PVector, pVector, [](float px, float py, float pz) { return array{px, py, pz}; });
 DECLARE_SOA_DYNAMIC_COLUMN(Eta, eta, [](float px, float py, float pz) { return RecoDecay::Eta(array{px, py, pz}); });
+DECLARE_SOA_DYNAMIC_COLUMN(Phi, phi, [](float px, float py) { return RecoDecay::Phi(px, py); });
```

Comment:
I think this causes an unneeded cast between double and float.
@aalkin What do you think about this?

Reply:
Promoting float to double is cheap and safe, so unless one would call this function always with double arguments, thus causing the demote, there is no problem.

Reply:
I guess that later phi is used as a float so we cast back.
What decides the type of a dynamic column?

Reply:
Return type of the lambda, in this case it is determined by the return type of `RecoDecay::Phi`, so it is double. Since dynamic columns are primarily for intermediate calculation it is fine to have them as double, it will not be demoted to float unless we explicitly assign it to a float variable.

Reply:
OK, I approve the PR.
I was thinking that most histograms are float, so at some point we go back to float. So does it really help to have double in between...